application/templates/static_site/measure.html

```diff
@@ -712,6 +712,10 @@ <h2 id="download-the-data" class="govuk-accordion__section-heading">
       }
     };
 
+    var chart_divs = document.getElementsByClassName('chart-and-footer');
+    for (var i=0, chart_div; chart_div = chart_divs[i]; i++) {
```

Comment:
Can we not do `for (chart_div in chart_divs)` like in the for loop above?

Reply:
Apparently not... because javascript/DOM is weird. In the loop above `dimensions` is defined as a list of dimension JSON objects.  What we have with `chart_divs` is a NodeList (I think), which doesn't iterate in the simple way you'd expect.  I tried it the obvious way and then had to look up why it was broken :man_shrugging: